apps/web/components/v2/eventtype/EventAppsTab.tsx

```diff
@@ -47,21 +48,25 @@ export const EventAppsTab = ({
   currency,
   eventType,
   hasGiphyIntegration,
+  giphyEnabled,
+  setGiphyEnabled,
+  paymentEnabled,
+  setPaymentEnabled,
+  rainbowEnabled,
+  setRainbowEnabled,
```

Comment:
I talked to @hariombalhara and this entire section needs to be extracted into packages/app-store.

each app should expose settings that can appear here.

we shouldn't create a state + component for each new app

cc @zomars for ideas

Reply:
Agreed, we already do some dynamic components in other areas